src/predefined.rs

```diff
@@ -178,31 +178,69 @@ impl PredefinedMenuItem {
 }
 
 /// Application metadata for the [`PredefinedMenuItem::about`].
-///
-/// ## Platform-specific
-///
-/// - **macOS:** The metadata is ignored.
-#[derive(PartialEq, Eq, Debug, Clone, Default)]
+#[derive(Debug, Clone, Default)]
 pub struct AboutMetadata {
     /// The application name.
     pub name: Option<String>,
     /// The application version.
     pub version: Option<String>,
+    /// The short version, e.g. "1.0"
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows / Linux:** This is ignored
+    pub short_version: Option<String>,
```

Comment:
Let's not add this one for the sake of cross-platform support and it doesn't seem to provide much value, like you can just use the `version` field and put whatever you want there.

Reply:
I see where you're coming from, but it is definitely not uncommon to see this field used across applications. In the interest of not restricting this functionality, maybe it would make sense to put it behind a `#cfg(target_os = "macos")` flag?

<img width="274" alt="image" src="https://github.com/tauri-apps/muda/assets/16108792/08a8cb1f-00f7-482e-abf1-140b77a575cd">

<img width="320" alt="image" src="https://github.com/tauri-apps/muda/assets/16108792/ee1b1251-b753-47d5-b4ef-0a47071e12b9">

Reply:
I think we can extend it to other platforms as well by using
```rs
fromat!("{}{}", version, short_version.map(|v| format!("({v})")).unwrap_or_default())
```

Reply:
Added in 4cd70e5